app/src/main/contrib/actions/contribTouchstonesActionCreators.ts

```diff
@@ -21,12 +21,7 @@ export const contribTouchstonesActionCreators = {
     setCurrentTouchstoneVersion(touchstoneVersionId: string) {
         return (dispatch: Dispatch<ContribAppState>, getState: () => ContribAppState) => {
             const touchstones = getState().touchstones.touchstones;
-            const versions = flatMap(touchstones, x => x.versions);
```

Comment:
Factored out this logic into a shared action creator, so that it can be re-used (re-use will appear in the next PR though!)